README.md

```diff
@@ -1,6 +1,6 @@
 # pt-tools
 
-This is a Go command line program that will allow listing and interacting with a Pairtree without knowing anything about the Pairtree’s internal structure. 
+This is a Go command line program that will allow listing and interacting with a Pairtree without knowing anything about the Pairtree’s internal structure. When using pt-tools, we highly suggest utilizing the `pairtree_prefix` file for schemes such as `ark:/`. When not using the`pairtree_prefix` file, denote the pairtree ID with `pt://`, so an ID should looke like `pt://ID`. This prefix will be stripped from an ID before interacting with  the pairtree. If no prefix is provided in the `pairtree_prefix` file and attached to the ID or `pt://` is not used, an error will be thrown.   
```

Comment:
I wonder whether, rather than referencing the `pairtree_prefix` file directly, we should just refer to "creating a pairtree with a prefix." That should be an argument to the future mkpt command. This just abstracts the prefix concept so that the pt-tools user doesn't have to know the implementation details of the pairtree (e.g., that there is an internal file for recording the prefix).

Reply:
That makes sense if we want to abstract the pairtree further from the user